torch/_torch_docs.py

```diff
@@ -635,19 +635,26 @@ def merge_dicts(*dicts):
             [1.0795, 2.1939]])
 """.format(**common_args))
 
-add_docstr(torch.as_tensor,
-           r"""
+add_docstr(torch.as_tensor, r"""
 as_tensor(data, dtype=None, device=None) -> Tensor
 
-Convert the data into a `torch.Tensor`. If the data is already a `Tensor` with the same `dtype` and `device`,
-no copy will be performed, otherwise a new `Tensor` will be returned with computational graph retained if data
-`Tensor` has ``requires_grad=True``. Similarly, if the data is an ``ndarray`` of the corresponding `dtype` and
-the `device` is the cpu, no copy will be performed.
+Converts :attr:`data` into a tensor.
+
+If :attr:`data` is already a tensor and :attr:`dtype` and :attr:`device` are
+None or match :attr:`data`'s dtype and device, then :attr:`data` is simply
+returned. Otherwise a new tensor is returned.
+
+If :attr:`data` is a NumPy array and the returned tensor has a matching dtype
```

Comment:
this is kind of backwards -- the previous paragraph talks about the inputs, whereas this talks about the output.  Is this because you don't want to explain the default dtype/device issue here?